alpaca_data_cleaned.json

```diff
@@ -80940,9 +80940,9 @@
         "output": "A successful strategy for improving user engagement should include components such as personalization, gamification, content curation, improved user interface, and data analysis. Personalization involves adapting content to the user\u2019s interests and preferences, which can help them feel connected to the product. Gamification refers to adding incentive features and rewards in the product to create a fun experience for users. Content curation is the process of organizing content in a way that's digestible and interesting to the user. An improved user interface can help streamline the user experience and make it more intuitive. Finally, data analytics can provide insights on how to optimize product features and operations."
     },
     {
-        "instruction": "Select 3 articles to read for the next meeting.",
-        "input": "A list of 10 potential articles (titles, authors, publication date and link).",
-        "output": "From the list of potential articles, the three articles to read for the next meeting are: \n\n- \"Ready, Steady, Socialize: Connecting during the Covid-19 Crisis\" by Caitlin Cowan (June 2, 2020, https://sloanreview.mit.edu/article/ready-steady-socialize-connecting-during-the-covid-19-crisis/) \n\n- \"The Role of Leadership in the New Normal\" by Jerald Sigler and Mike Myatt (June 8, 2020, https://www.forbes.com/sites/advisors/2020/06/08/the-role-of-leadership-in-the-new-normal/#4d4b1d795eeb/) \n\n- \"How to Adapt to an Uncertain Economy\" by Minouche Shafik (June 26, 2020, https://www.weforum.org/agenda/2020/06/how-to-adapt-to-an-uncertain-economy/)."
+        "instruction": "Construct a web address for a book recommendation website.",
```

Comment:
This is duplicated. There seems to be a bug with "Reset to begining." Will look into it.

Reply:
I found the problem. When the user refreshes the page, the fields gain their initial value: the first match from the dataset. But in the backend, they are still on the item you last viewed. If you then perform any action, it will save it as if you changed the current item, since that is essentially what you did.
Not sure how to fix this since I don't see any event to capture refresh of the page inside Gradio. It's not that big of a problem, I guess.